### DISCUSSION TITLE
Flexible DR indication?

### DISCUSSION THREAD

**Discussion by ndrwstn** [richardwilkes/gcs#415] [Q&A] [answered]
GURPS uses a '*' in DR to represent flexible DR (B282), but I haven't found a way to appropriately represent this on GCS. The new split DRs don't exactly fit the bill. Am I missing something or should I create a feature request? Thanks.

**Comment by richardwilkes** [comment, marked as answer]
This is typically done via a note on the item in GCS. If you look at the armor in the Basic Set Equipment, for example, you'll note a number of them that have a note that says "Flexible" or "Flexible, concealable", etc.

**Reply by ndrwstn** [reply]
Gotcha. Are you opposed to any changes that might reflect flexible or concealable in the DR section? Is that something that can be done?

**Reply by richardwilkes** [reply]
I'm not opposed to such things. However, I don't know if GURPS has been consistent in the area (there are a number of different symbols that they use for things, and I think I've sometimes seen contradictory definitions depending upon the source book), which is why I've generally avoided that.
